Add styles to support printing the tutorial

sruggier opened the pull request:
In particular, this makes it possible to save an offline copy of the tutorial to PDF or paper, or any blog posts of interest.

Darksonn commented:
Thanks. I tried it and it looks ok. However, there's a CI failure due to the "prettier" formatter.

sruggier commented:
Thanks, it should be fixed now. I also added a bit of detail to the first commit's subject and description.

sruggier commented:
@Darksonn Would you mind re-approving the CI workflow? It should pass now.

mox692 commented:
@sruggier I just re-ran tests. (It seems that some tests are still failing, though)

sruggier commented:
@mox692 Thanks! I've rebased on the master branch, which seems to have [a fix](https://github.com/tokio-rs/website/commit/abc0d296601794d85fe1e20280543266f2570552) for the failing tests.

mox692 commented on styles/styles.scss:
```diff
@@ -79,6 +79,7 @@ $footer-background-color: black;
 $footer-color: white;
 
 @import "../node_modules/bulma/bulma.sass";
+@import "../node_modules/bulma-print/bulma-print";
```
Apparently, what this import refers to is just [a few lines of CSS](https://github.com/suterma/bulma-print/blob/7389bd6b28798c89163ed8886eaed0162b4aee96/sass/helpers.scss#L6-L40), so I think we can simply paste it inline in `styles/styles.scss` rather than adding bulma-print. What do you think?

(Also, it seems that `bulma-print` library hasn't been updated for some time.)

mox692 reviewed (commented):
Sorry for the delay, I left a comment about dependency addition, but otherwise looks good to me!

sruggier committed:
Add styling specific to the printed version

This adds printing-specific styles, which hide the navigation elements,
remove excess whitespace, wrap code blocks, and avoid breaking them
across separate pages, if possible.

This makes it possible to export the website and documentation to PDF or
paper, for offline reading.

sruggier committed:
Add styling specific to the printed version

This adds printing-specific styles, which hide the navigation elements,
remove excess whitespace, wrap code blocks, and avoid breaking them
across separate pages, if possible.

This makes it possible to export the website and documentation to PDF or
paper, for offline reading.
